### ISSUE TITLE
Burp-Plugin - Enhancements or Issues

### ISSUE THREAD

**Issue by dragos-sh** [DefectDojo/Burp-Plugin#17]
Hi there, 

So, we have played for a while with the Burp Plugin and somehow made it work. At this point, we have the following questions: 

1. DefectDojo does have API v1/v2. Why is the plugin limited to work only with an API v1 key? 
2. We have looked at the demo movie for 100 times probably, and under the current plugin build version, we are not getting the same fields autocompleted. See the attachment where everything highlighted as part of your demo clip is not there for us. 

![image](https://user-images.githubusercontent.com/49570819/59078912-64449900-8935-11e9-802d-998f7c505e00.png)

3. When we are using the "send to DefectDojo" option, we have not seen things happen. We've played around, and at some point, such details started to get through. 

![image](https://user-images.githubusercontent.com/49570819/59079076-17ad8d80-8936-11e9-9d2a-789952376611.png)

Now, our question is what those entries have to represent or contain[currently they are completely emty]? Should we edit each of those fields with the proper vuln description, etc and submit them manually, or? How the process should work and what we have to expect when we are clicking "Sent to Defect Dojo."?

4. There is a second option too, "Send report to Defect Dojo." It never worked out for us. "Import possibly failed." What should this option do? 

If you guys are able to provide more details into the current demo movie like what should happen at the defectdojo platform level after a user is using the burp plugin trying to push some data into the platform, that would be great. 

As a final note, apologies for any rooky-ish questions detailed above. If there are any docs around which do have the answers to our questions that would be appreciated too. 

Thanks!

**Comment by adracea** [comment]
Hello,
1. We use APIv1 for this 
2. The UI side looked a bit cluttered and that side looked somewhat redundant once the context menus were working , so they were removed to make it look less visually disturbing .
3. Depending on the issue you are trying to send , some issues from other extensions don't respect the burp format itself(some fields might be missing , weirdly encoded data etc.) and I can't account for these . There should also be a stacktrace of the import failing in the Extender tab under the plugin's entry .
4. Hmm...Again , when you encounter this , leave a stacktrace , I need to see these in order to be able to figure out what is happening .
Cheers ,
Alex

**Comment by dragos-sh** [comment]
@adracea solid copy. we will try to picture on something 4 you asap.

**Comment by dragos-sh** [comment]
Right. While we have tried to record a PoC movie we started to see what you have said, At this point, we can send to DefectDojo only the default burp findings. Anything else reported by third-party extensions just simply won't work. Personally, I won't be going to push to have a workaround for those because pb is a pain in the neck as everyone is free to build whatever extensions and use them. We will close this one now. Thanks for you jumped in. Cheers.
